doc/lspconfig.txt

```diff
@@ -242,34 +242,37 @@ The `configs` module is a singleton where configs are defined. The schema for
 validating using `vim.validate` is:
 >
     configs.SERVER_NAME = {
-      default_config = {'t'};
-      on_new_config = {'f', true};
-      on_attach = {'f', true};
-      commands = {'t', true};
-      docs = {'t', true};
+      default_config = {'t'},
+      on_new_config = {'f', true},
+      on_attach = {'f', true},
+      commands = {'t', true},
+      docs = {'t', true},
     }
 <
 where the structure of the docs table is as follows:
 >
     docs = {
-      description = {'s', true};
-      default_config = {'t', true};
+      description = {'s', true},
+      default_config = {'t', true},
     }
 <
-`commands` is a map of `name:definition` key:value pairs, where `definition`
-is a list whose first value is a function implementing the command, and the
-rest are either array values which will be formed into flags for the command,
-or special keys like `description`. Example:
+`commands` is a list of tables where each table consists of the fields
+`name`, `command`, and `opts`. See |nvim_create_user_command| for more
+details.
```

Comment:
why was it changed from a map?

Reply:
As I understand it, this is to make the interface identical to `nvim_create_user_command`